api/hardwaremanagement/v1alpha1/node_pools.go

```diff
@@ -91,6 +91,9 @@ type NodePoolStatus struct {
 // +kubebuilder:resource:shortName=np
```

Comment:
Needs to be removed?

Reply:
Oh, I did not see that one. I'll remove it, yes. Thank you.